Core/OxidEE_Events.php

```diff
@@ -160,38 +157,26 @@ private static function _createOrderTable()
     private static function _createOrderTransactionTable()
     {
         $sQuery = "CREATE TABLE IF NOT EXISTS `wdoxidee_ordertransactions`(
```

Comment:
For me, this table is also created even if it already exists.

Test case: activate plugin → add an entry to `wdoxidee_ordertransactions` → disable and re-activate plugin → entries are gone